### DISCUSSION TITLE
User submitting inappropriate named resources to multiple galaxies

### DISCUSSION THREAD

**Discussion by ghost** [pwillworth/galaxyharvester#55] [General]
http://galaxyharvester.net/user.py/esbeltatamaral

Not sure if a simple ban would do it (I assume he'd just create a new account)?

**Comment by ghost** [comment]
@pwillworth

**Comment by pwillworth** [comment]
Hi @jawsont , yeah there's not much to be done other than remove the bad resources.  As people do that, it reduces his reputation and removes his ability to make entries with that account.  He can just create another one though as it looks like he as already done at least once.  Eventually he will get bored.

**Reply by ghost** [reply]
okies (and tyvm for the reply)

**Reply by ghost** [reply]
I hate to waste more of your time on it, but somehow @pwillworth he's gotten around the negative reputation scores for posting. One of his accounts has -10k reputation and he's still posting.  O.o

**Reply by pwillworth** [reply]
Wow... looking at the code it seems that I never even implemented the reputation check for adding stuff at all, just all the other abilities.  I'll have to work on implementing that, might take a day or two.

**Reply by ghost** [reply]
Again - I just appreciate what you do. 99.99% of the time users are decent and reputation isn't needed. I vaguely recall you trying to implement it maybe 5-6y ago, but I feel like there maybe was an issue that locked some folks out and I thought you temporarily turned it off? But it's a hazy memory for sure.

**Reply by pwillworth** [reply]
Oh yeah, you are spurring a fuzzy memory for me too.  I'll have to dig a bit and see if I can find the why on that.  Hopefully it's something that doesn't apply anymore or can be worked around.
